flowise-embed/src/components/Bot.tsx

```diff
@@ -1184,6 +1161,7 @@ export const Bot = (botProps: BotProps & { class?: string }) => {
         : uuidv4();
 
       setChatId(newChatId);
+      setLocalStorageChatflow(props.chatflowid, chatId());
```

Comment:
Talvez seja melhor usar `newChatId` ao invés de `chatId()` aqui, dada a natureza assíncrona do React e à (ainda que pequena) possibilidade de `chatId()` ainda estar com o valor antigo quando for chamado.